feature-configs/demo/sriov/sriov-networknodepolicy-dpdk.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: dpdk-network-node-policy-dpdk
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+    pfNames:
+      - eno2
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+    node-role.kubernetes.io/worker-cnf: ""
```

Comment:
This selector can be one with `and` (or probably sticking to `worker-cnf` is fine)

Reply:
sure